File: OCH_Tao/tessoku-book/A03.js
const [[, K], [...P], [...Q]] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n").map(x => x.split(" ").map(y => parseInt(y, 10)));
for (const i of P) {
  for (const j of Q) {
    if (i + j === K) {
      console.log("Yes");
      process.exit();
    }
  }
}
console.log("No");

File: OCH_Tao/tessoku-book/A03.py
N, K = map(int, input().split())
P = list(map(int, input().split()))
Q = list(map(int, input().split()))
for i in P:
  for j in Q:
    if i+j == K:
      print("Yes")
      break
  else:
    continue
  break
else:
  print("No")
